ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/ParametersTable.scala

```diff
@@ -41,8 +32,16 @@ private[appendonlydao] object ParametersTable {
   private val LedgerEndOrBeforeBeginParser: RowParser[Offset] =
     LedgerEndParser.map(_.getOrElse(Offset.beforeBegin))
 
-  private val LedgerEndParserAndSequentialIdParser =
-    long(LedgerEndSequentialIdColumnName).?.map(_.getOrElse(EventSequentialId.BeforeBegin))
+  private val LedgerEndOffsetAndSequentialIdParser =
+    (offset(LedgerEndColumnName).? ~ long(LedgerEndSequentialIdColumnName).?)
+      .map(SqlParser.flatten)
+      .map {
+        case (Some(offset), Some(seqId)) => (offset, seqId)
+        case (Some(offset), None) => (offset, EventSequentialId.beforeBegin)
```

Comment:
Is this a real possibility?

Reply:
After a first configuration update? or package upload, no?

Reply:
Makes sense, thanks